dev/change_version.py

```diff
@@ -237,6 +238,9 @@ def change_version(reef_home, new_version, pom_only):
         change_assembly_info_cs(reef_home + "/lang/cs/Org.Apache.REEF.Bridge/AssemblyInfo.cpp", new_version)
         print reef_home + "/lang/cs/Org.Apache.REEF.Bridge/AssemblyInfo.cpp"
 
+        change_assembly_info_cs(reef_home + "lang/cs/Org.Apache.REEF.ClrDriver/AssemblyInfo.cpp", new_version)
```

Comment:
The updated script doesn't work; it needs `/` before `lang`, same as other entries.